guides/source/developers/events/overview.html.md

```diff
@@ -0,0 +1,55 @@
+# Events
+
+The `Spree::Event` module allows to fire and subscribe events inside Soldius codebase and extensions. 
+
+The module can use different adapters that actually manage events at low level, the default adapter is Rails `ActiveSupport::Notification`. Future releases may include other adapters. 
+
+Among other uses, Solidus codebase uses events in order to send confirmation emails when an order is finalized, or again to send emails when an order is refunded successfully. 
+
+Events make easy extending Solidus with custom behavior. For example, if besides the standard email you also want to send a SMS text message to the customer when a order is finalized, this pseudo-code may do the trick:
+
+```ruby
+Spree::Event.subscribe 'order_finalized' do |event|
+  order = event.payload[:order]
+  SmsLibrary.deliver(order, :finalized)
+end
+```
+
+## Changing the adapter 
+
+The adapter can be changed using this code, for example in a initializer:
+
+```ruby
+Spree::Config.events.adapter = "Spree::EventBus.new"
+```
+
+## Subscribing to events
+
+`Spree::Event.subscribe` allows to subscribe to a certain event. The event name is mandatory, the optional block will be executed everytime  the event is
```

Comment:
Two things in here:

* There's an trailing whitespace after `everytime`
* `fired:` is on a new line, I think it should be on the same line as the paragraph above

Reply:
👍